MoneyDataType/Abstractions/ICurrencyProvider.cs

```diff
@@ -1,30 +1,27 @@
 using System.Collections.Generic;
 
-namespace Money.Abstractions
+namespace Money.Abstractions;
+
+/// <summary>
+/// A service that provides a set of currencies.
+/// </summary>
+public interface ICurrencyProvider
 {
     /// <summary>
-    /// A service that provides a set of currencies
+    /// Gets the list of currencies.
     /// </summary>
-    public interface ICurrencyProvider
-    {
-        /// <summary>
-        /// The list of currencies
-        /// </summary>
-        IEnumerable<ICurrency> Currencies { get; }
+    IEnumerable<ICurrency> Currencies { get; }
 
-        /// <summary>
-        /// Finds the currency object for the provided symbol 
-        /// </summary>
-        /// <param name="isoCode">The three-letter ISO 4217 code for the currency</param>
-        /// <returns>The currency object</returns>
-        ICurrency GetCurrency(string isoCode);
+    /// <summary>
+    /// Finds the currency object for the provided symbol.
+    /// </summary>
+    /// <param name="isoCode">The three-letter ISO 4217 code for the currency.</param>
+    /// <returns>The currency object.</returns>
```

Comment:
That's redundant, we already have that in the summary. I don't think we need that <returns> at all, it provides no new information
Also now that I read it, the description not correct. A symbol would be like $ or € but we provided an ISO code USD and EUR.

Reply:
Repetition is not a bad thing in XML docs, because every element belongs to a different part of the method signature. Look:

![image](https://user-images.githubusercontent.com/164360/165626900-1e0b9175-5710-488b-9a3d-42f0e4e5d558.png)

![image](https://user-images.githubusercontent.com/164360/165626917-ed836c23-9d08-45fc-ad1d-70ef2a5c6272.png)

When you look at the `Returns` block, my suggested version reads a lot more consistent, imo. You can skip reading the summary above, which may contain more details on the hows and whys of the method, that you might just not care about right now.

Reply:
Most of our own functions and many of the functions in this library have no `<returns>` documentation, so if this was important we'd already be in big trouble.
Also I disagree on repetition not being a bad thing: repetition means you have to maintain the same facts in different places, so any change has a higher danger of desync and contradition.